Sources/Rendering/Core/Texture/index.js

```diff
@@ -69,6 +72,8 @@ export function extend(publicAPI, model, initialValues = {}) {
     'image',
   ]);
 
+  model.viewSpecificProperties = {};
```

Comment:
Don't you check first if that `viewSpecificProperties` was given to you in the constructor? Otherwise here, you will always reset it.

If would do that instead

```
if (!model.viewSpecificProperties) {
 viewSpecificProperties = {};
}
```